CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.4.0 TBD
+
+* Extract SO files from bugsnag-android artifact
```

Comment:
This is a more significant change than these notes imply, and this may need to be more verbose to convey that and ensure that anyone who upgrades bugsnag-android without bugsnag-android-gradle-plugin can see that the latest version will fix their problem.

```suggestion
This release is companion update for bugsnag-android v4.15.0, which supports detecting and reporting C/C++ crashes without a separate library. 

Extract shared object C libraries (*.so files) from bugsnag-android artifact to support using the Native API
```

\* Didn't cross-link "Native API" since the docs might move.

Reply:
Sounds good. We should add something to the bugsnag-android notes too when that's released, as people are more likely to read those IMO.